### DISCUSSION TITLE
[QUESTION] Assets events

### DISCUSSION THREAD

**Discussion by RaresVlaiduc** [GrapesJS/grapesjs#4068] [Q&A]
Hello all!

I have a question regarding the assets events.
The version of grapesjs I'm currently using is 0.17.27.
I need to upload the assets on a specific server and I've seen in the documentation that there are some multiple events that can be used.
For example I saw 'assets:upload:start' but apparently this event does not exist in this version. The only events I gen on editor.AssetsManager.events are the following:

"asset:add"
"asset"
"asset:close"
"asset:custom"
"asset:open"
"asset:remove"
"asset:remove:before"
"asset:select"
"asset:update"

Is this a version issue? I've looked on the releases but I couldn't find anything changed after 0.17.27.

Thank you!

**Comment by artf** [comment]
Yeah seems like I forgot to include them in `editor.AssetsManager.events` post-refactoring (which honestly I don't even consider it as a public property) but I'd expect `assets:upload:*` to work properly as per documentation.
